src/sample/simple.py

```diff
@@ -7,4 +7,4 @@ def subtract_one(number):
 
 
 def inverse(number):
-    return 1 // number
+    return 1 / number
```

Comment:
Could you maybe add a docstring to this function. This makes it easier for new users to apply it.

Reply:
done